Automattic/jetpack
# Add/openai placeholder prompts

Fixes https://github.com/Automattic/jetpack/issues/28325

## Proposed changes:

Add prompt suggestions for AI Image

## Testing instructions:

- Deploy this to your sandbox.
- Sandbox a site
- Create an AI Image
- You should see a placeholder text. Test that it behaves accordingly.
- When clicking on submit it prompts with the placeholder
- 
<img width="762" alt="Screenshot 2023-01-13 at 11 37 38" src="https://user-images.githubusercontent.com/790558/212300720-810e05cb-b212-4a9e-8cb1-ffd4eed7451b.png">

## Examples of the prompts results

<img width="708" alt="Screenshot 2023-01-13 at 17 20 56" src="https://user-images.githubusercontent.com/790558/212371239-b5ab4767-823a-4de1-9919-edc9c4be1136.png">
 <img width="708" alt="Screenshot 2023-01-13 at 17 21 02" src="https://user-images.githubusercontent.com/790558/212371248-7d2ad7fb-260c-43ff-84d3-993269d27e08.png">
<img width="705" alt="Screenshot 2023-01-13 at 17 21 10" src="https://user-images.githubusercontent.com/790558/212371255-96592366-6b48-49af-92e7-dacaa399103c.png">
<img width="707" alt="Screenshot 2023-01-13 at 17 21 17" src="https://user-images.githubusercontent.com/790558/212371257-72e63d83-4bb5-4c53-a98a-acd2ec848cdb.png">


<img width="709" alt="Screenshot 2023-01-13 at 17 20 48" src="https://user-images.githubusercontent.com/790558/212371214-43561b86-bf2d-4053-979e-3aab09c274dd.png">
